e2e/src/step_definitions/media-viewer.ts

```diff
@@ -113,6 +119,13 @@ Then('I expect bookmark to be added to the existing list', async function () {
   this.attach(screenshots, 'image/png');
 });
 
+Then('I expect to see the document should be downloaded', async function () {
```

Comment:
we're not actually checking whether the document has been downloaded

Reply:
When the script clicks download option,  the document is downloading and at that stage we are capturing screenshoot.